lib/cog_api/http/relay_groups.ex

```diff
@@ -60,12 +60,40 @@ defmodule CogApi.HTTP.RelayGroups do
     update_membership(relay_group_id, relay_id, :remove, endpoint)
   end
 
+  def assign_bundle(%{name: name}, %{bundle: bundle_name}, %Endpoint{}=endpoint) do
+    with {:ok, bundle} <- Base.get_by(endpoint, "bundles", name: bundle_name)
+      |> ApiResponse.format(%{"bundle" => CogApi.Resources.Bundle.format}),
+      {:ok, relay_group} <- Base.get_by(endpoint, "relay_groups", name: name)
+      |> ApiResponse.format(%{"relay_group" => RelayGroup.format}),
+      do: update_assignments(relay_group.id, bundle.id, :add, endpoint)
+  end
+  def assign_bundles(relay_group_id, bundle_id, %Endpoint{}=endpoint) do
+    update_assignments(relay_group_id, bundle_id, :add, endpoint)
+  end
+
+  def unassign_bundle(%{name: name}, %{bundle: bundle_name}, %Endpoint{}=endpoint) do
+    with {:ok, bundle} <- Base.get_by(endpoint, "bundles", name: bundle_name)
```

Comment:
It seems like the first few lines of the `assign` and `unassign` are nearly the same, if not identical. Could we pull those out to some shared function?